packages/regl-worldview/src/commands/Text.js

```diff
@@ -50,11 +57,24 @@ function insertGlobalCss() {
   cssHasBeenInserted = true;
 }
 
+function isColorDark({ r, g, b }: Color): boolean {
+  // ITU-R BT.709 https://en.wikipedia.org/wiki/Rec._709
+  // 0.2126 * 255 * r + 0.7152 * 255 * g + 0.0722 * 255 * b
+  const luma = 54.213 * r + 182.376 * g + 18.411 * b;
+  return luma < BRIGHTNESS_THRESHOLD;
+}
+
+function isColorEqual(a: Color, b: Color): boolean {
+  return a.r === b.r && a.g === b.g && a.b === b.b && a.a === b.a;
```

Comment:
use lodash's `isEqual` instead? Does deep comparison of objects.

Reply:
Since we are need the `Text` component to be as performant as possible (that's the reason we created `TextElement ` instead of using React's DOM update), it would be better to not introduce `isEqual` to do deep object comparison as we could just simply compare the 4 color fields